plugins/plugin-factory-github/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I thought about that but I was not sure as it's linked in some way to github but not using the current stuff in this directory
But I'm ok to change